apps/AEPSampleApp/extensions/CampaignClassicView.tsx

```diff
@@ -0,0 +1,42 @@
+import {CampaignClassic} from '@adobe/react-native-aepcampaignclassic';
+import React from 'react';
+import {Button, ScrollView, Text, View} from 'react-native';
+import styles from '../styles/styles';
+import {NavigationProps} from '../types/props';
+
+function CampaignClassicView({navigation}: NavigationProps) {
+  const extensionVersion = async () => {
+    const version = await CampaignClassic.extensionVersion();
+    console.log(`AdobeExperienceSDK: Campaign Classic version: ${version}`);
+  };
+
+  const registerDeviceWithToken = () =>
+    CampaignClassic.registerDeviceWithToken('myToken', 'myUserKey');
+  const trackNotificationClickWithUserInfo = () =>
+    CampaignClassic.trackNotificationClickWithUserInfo({userId: '1'});
+  const trackNotificationReceiveWithUserInfo = () =>
+    CampaignClassic.trackNotificationReceiveWithUserInfo({userId: '1'});
```

Comment:
can we pass in `_mId` and `_dId` in both these examples?
`CampaignClassic.trackNotificationReceiveWithUserInfo({`_mId`: `12345`, `_dId`: `testDId`})`